pyproject.toml

```diff
@@ -0,0 +1,131 @@
+[build-system]
+requires = ["wheel", "setuptools>=61.2", "setuptools-scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "aehmc"
+authors = [
+    {name = "aesara-devs", email = "aesara.devs@gmail.com"}
```

Comment:
Minor aesthetic point: I think we can change the name to "Aesara developers" both here and in the Aesara repo.